gamemode/config/config.lua

```diff
@@ -325,6 +325,8 @@ GM.Config.lockdownsound = "npc/overwatch/cityvoice/f_confirmcivilstatus_1_spkr.w
 -- The skin DarkRP uses. Set to "default" to use the GMod default derma theme.
 GM.Config.DarkRPSkin = "DarkRP"
 GM.Config.currency = "$"
+GM.Config.currencyDecimalSeparator = "."
```

Comment:
For float reasons, I don't think it's wise to have options related to decimal numbers. DarkRP officially doesn't support decimal numbers. The reason for that is float inconsistencies. It's quite hard to avoid rounding errors and money amounts looking like `$2.000000000001`